communication/src/dtls_protocol.h

```diff
@@ -115,6 +115,13 @@ class DTLSProtocol : public Protocol
 			}
 			return r;
 		}
+		case ProtocolCommands::MOVE_CONNECTION: {
+			LOG(INFO, "Moving connection (session move + ping)");
+			if (!channel.command(Channel::MOVE_SESSION)) {
```

Comment:
Any reason not to return the actual error code instead of `UNKNOWN` if this fails?

Reply:
:shrug: fixed. And added one session use.